joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/passes/PhpTypeRecoveryPassTests.scala

```diff
@@ -370,12 +370,12 @@ class PhpTypeRecoveryPassTests extends PhpCode2CpgFixture() {
 
     "be properly resolved when called with $this" in {
       val List(fooCall) = cpg.method("bar").ast.isCall.take(1).l
-      fooCall.methodFullName shouldBe "ClassA->foo"
+      fooCall.methodFullName shouldBe "ClassA.foo"
     }
 
     "be properly resolved when called through class with known type" in {
-      val List(fooCall) = cpg.method("baz").ast.isCall.filter(_.code == "$a->foo()").take(1).l
-      fooCall.methodFullName shouldBe "ClassA->foo"
+      val List(fooCall) = cpg.method("baz").ast.isCall.filter(_.code == "$a.foo()").take(1).l
```

Comment:
Why would you also change the code property? Please leave that as it was to reflect the actual code.